MessageBird/Resources/Voice/WebhookLists.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using MessageBird.Objects.Voice;
+
+namespace MessageBird.Resources.Voice
+{
+    public class WebhookLists : VoiceBaseLists<Webhook>
```

Comment:
I'm wondering if we need this class and even the WebhookList. In the recordings, I created the RecordingLists because we had to override the `Uri` property. But if you're not overriding anything here, maybe we don't need it at all? Can you please check?